rocAL/rocAL/source/pipeline/master_graph.cpp

```diff
@@ -793,15 +793,88 @@ MasterGraph::copy_out_tensor(void *out_ptr, RocalTensorFormat format, float mult
     #endif
                         }
                     }
-                    else if(output_data_type == RocalTensorDataType::FP16) {
+                    else if(output_data_type == RocalTensorDataType::FP16) 
+                    {
```

Comment:
Align { wrt other code in this function.

Reply:
Leaving it as it is